client/src/pages/Home.js

```diff
@@ -49,6 +58,18 @@ const SectionHeadline = styled.h4`
   margin: 1.5rem auto 1rem;
 `;
 
+const LogoutButton = styled.button`
+  border: none;
+  border-radius: var(--border-radius);
+  background-color: #cc1c1c;
```

Comment:
you mix var color and hex code here. maybe create a new var instead of using hex code?